packages/server/graphql/getRateLimiter.ts

```diff
@@ -1,9 +1,9 @@
-import RateLimiter from './RateLimiter'
+import {InMemoryRateLimiter} from './InMemoryRateLimiter'
 
-let rateLimiter: RateLimiter
+let rateLimiter: InMemoryRateLimiter
```

Comment:
I found the name InMemoryRateLimiter a bit clearer, but I can change this back to just `RateLimiter` if you'd like.